### PR TITLE
Draft dice and points

### PR DESCRIPTION
**Pull request by agude**
Starting to write out the mechanics section.

I have an intro and an example... But they both need a lot of work and the example needs a home.

### CODE REVIEW

File: chapters/04-dice-and-points.md [hunk 1]
```diff
@@ -1,11 +1,27 @@
 # Dice and Fate Points
```
Comment (agude) [inline chapters/04-dice-and-points.md:1]: Better names:

- Deciding What Happens
- Narrative Control
- Game Mechanics
- Mechanics
- Taking Action
- Resolving Conflicts

?

I actually don't hate it as is...

File: chapters/04-dice-and-points.md [hunk 2]
```diff
@@ -1,17 +1,33 @@
...
+- Narrative Control
+- Game Mechanics
+- Mechanics
+- Taking Action
+- Resolving Conflicts
 
-Fate points are a currency of narrative control. You can spend a fate point for a bonus on a roll, to modify a scene, or X
+-->
+
+Dice rolls in MiniFate should be used sparingly. They add an element of
```
Comment (agude) [inline chapters/04-dice-and-points.md:14]: Maybe this is actually:

> Dice rolls add... But they should be used sparingly; PCs are assumed...

File: _chapters/04-dice-and-points.md [hunk 3]
```diff
@@ -2,18 +2,33 @@
...
+- Resolving Conflicts
 
-Fate points are a currency of narrative control. You can spend a fate point for a bonus on a roll, to modify a scene, or X
+-->
+
+Dice rolls add an element of tension and unpredictability that keeps your
+story exciting; without them you might as well just write a novel! But they
+should be used sparingly. PCs are competent heroes especially in the things
+their aspects imply they should be good at! If their is no interesting result
+of failing a roll, assume the action happens and move on with the story.
```
Comment (charles-uno) [inline _chapters/04-dice-and-points.md:20]:
```suggestion
MiniFate uses dice to add an element of tension and unpredictability to the story.
Without them, you might as well be writing a novel!
That said, you don't need to roll for every little thing. 
As a general rule, the GM should prompt players for a roll in situations where a competent hero might reasonably fail *and* failure adds an interesting twist to the story.
```

Comment (charles-uno) [reply _chapters/04-dice-and-points.md:20]: I worry about going too deep on the intro blurb for this chapter, and leaving nothing for the skill rolls section to say

Comment (charles-uno) [reply _chapters/04-dice-and-points.md:20]: How a bout something like:

Intro blurb:

> MiniFate uses dice to add an element of unpredictability to the story.
> Without them, you might as well be writing a novel!
> That said, it's not all up to chance.
> Players can spend fate points to re-roll the dice at dramatically appropriate moments, or even invent new > aspects about the world.
> But watch out---the GM has fate points as well, which they can use to complicate the PCs' lives at just the wrong moment.

Skill Rolls:

> Dice are an important part of MiniFate, but that doesn't mean you need to roll for every little thing.
> As a general rule, the GM should prompt players for a roll in situations where a competent hero might reasonably fail *and* failure adds an interesting twist to the story.
> For example, a ~~Swashbuckling Genius~~ probably doesn't need to roll to fend off an attack by a ~~Nameless Deck-Hand~~... but she might roll `Fight` to subdue him before his friends escape, or `Network` to recognize him.

**Review by charles-uno — APPROVED** [review]

**Comment by agude** [comment]
@charles-uno Want to take a last look? Or just merge and we'll iterate as I write more.

**Review by charles-uno — APPROVED** [review]
👍

**Pull request merged**